### PR TITLE
lxqt-meta and friends: Version bump to 0.11.1, 0.11.2, and/or 0.11.3.

### PR DESCRIPTION
**Pull request by Chiitoo**
I hope I got all the required packages, without too many mistakes sneaking in.  I left out 'lxqt-l10n' for now, as it needs some more thought since it's installing conflicting files with 'x11-misc/obconf-qt'.

Was already about half way in when I realised most of them were still using EAPI 5, so that's something to take care of at some point as well...

I checked with upstream IRC-channel that it's OK to have some of the packages with higher version numbers than others, though I'm sure at some point(s) it must fail.  Should probably make that picture a bit more clear from what it is now.

One may notice that some of these now have '>=dev-util/lxqt-build-tools-0.3.2' while others have 'dev-util/lxqt-build-tools'.  It seems that building is OK with 0.2.0 for the 0.11.1 packages, but not those that go beyond that.

In hindsight, I could have probably left 'dev-util/lxqt-build-tools' out of 'lxqt-meta', since it doesn't seem like it would be something for the RDEPEND...

Now I think I need a git... I mean a nap.

### CODE REVIEW

**Comment by Chiitoo** [comment]
Added 'media-gfx/lximage-qt: Version bump to 0.5.1.'

**Comment by monsieurp** [comment]
@Chiitoo: Hi! Could you get someone from the lxqt team to review your PR please? Thanks!

**Comment by Chiitoo** [comment]
I'll do what I can.

Thanks, monsieurp!

**Comment by Chiitoo** [comment]
Updated 'x11-misc/sddm' requirement to >= 0.14.0, and 'dev-libs/libqtxdg' requirements to >=2.0.0.

**Comment by Chiitoo** [comment]
OK so, we might need someone brave enough from proxy-maint to look into this (cough @monsieurp), since the LXQt project members are not having the time for it.  :]

File: lxqt-base/libsysstat/libsysstat-0.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/libsysstat/libsysstat-0.3.3.ebuild]: Can you bump it to EAPI 6?

File: lxqt-base/lxqt-about/lxqt-about-0.11.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-about/lxqt-about-0.11.1.ebuild]: Ditto.

File: lxqt-base/lxqt-admin/lxqt-admin-0.11.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-admin/lxqt-admin-0.11.1.ebuild]: Here too.

File: lxqt-base/lxqt-common/lxqt-common-0.11.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-common/lxqt-common-0.11.2.ebuild]: Same deal.

File: lxqt-base/lxqt-globalkeys/lxqt-globalkeys-0.11.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-globalkeys/lxqt-globalkeys-0.11.1.ebuild]: Again.

File: x11-misc/obconf-qt/obconf-qt-0.11.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline x11-misc/obconf-qt/obconf-qt-0.11.1.ebuild]: And again.

File: lxqt-base/lxqt-meta/lxqt-meta-0.11.1.ebuild [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-meta/lxqt-meta-0.11.1.ebuild]: .

File: lxqt-base/lxqt-notificationd/lxqt-notificationd-0.11.1.ebuild [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-notificationd/lxqt-notificationd-0.11.1.ebuild]: =/

File: lxqt-base/lxqt-openssh-askpass/lxqt-openssh-askpass-0.11.1.ebuild [hunk 9]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-openssh-askpass/lxqt-openssh-askpass-0.11.1.ebuild]: :O

File: lxqt-base/lxqt-panel/lxqt-panel-0.11.1.ebuild [hunk 10]
```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-panel/lxqt-panel-0.11.1.ebuild]: Please.

File: lxqt-base/lxqt-policykit/lxqt-policykit-0.11.1.ebuild [hunk 11]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-policykit/lxqt-policykit-0.11.1.ebuild]: Rest.

File: lxqt-base/lxqt-powermanagement/lxqt-powermanagement-0.11.1.ebuild [hunk 12]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (monsieurp) [inline lxqt-base/lxqt-powermanagement/lxqt-powermanagement-0.11.1.ebuild]: Okay you get the idea. Please bump all EAPI 5 ebuilds to version 6.

**Comment by monsieurp** [comment]
:+1:

**Comment by Chiitoo** [comment]
I was almost certain someone would ask for that.  :]

(I would have probably worked on it, had I noticed they were still EAPI 5 early enough...)

I'm also currently going through them with NP-Hardass, and some other improvements will likely be on their way from that.

Thank you!

**Comment by Chiitoo** [comment]
Just an update on what's going on with this:

I've recently been a bit busy with other things, and now my eye is acting up while there is quite a bit of things to be improved upon here.

Some things I've been doing are bumping EAPI, adjusting versions for dependencies, figuring out what to do with 'lxqt-common' since upstream has deprecated it before releasing the packages with its files (right now thinking of going with 0.11.1 instead of 0.11.2, since the former can be downloaded from their domain, while 0.11.2 is only in their current main repository, GitHub, but with the repository having been renamed to 'lxqt-common-deprecated--do-not-use-anymore-'... though of course that could be worked in just as well).

Also working out the conflicts between 'obconf-qt' and 'lxqt-l10n' (https://bugs.gentoo.org/601844) by using 'PULL_TRANSLATIONS'.  After that I was wondering if it's going to be needed for all the ebuilds, but the more recent ones already have it... so that is as they say, that!

I'll hopefully manage to get through the /rest/ soon enough, unless someone else will.  :]

**Comment by Chiitoo** [comment]
Actually managed to go through them after my last message...

There may be some obvious errors even, and I did not try building everything yet, and as such, I would probably not recommend merging just yet even if things looked good!

**Comment by Chiitoo** [comment]
Everything seems to build OK, though I did not go as crazy as to test all the possible USE-flag combinations.

I'll probably remove the cmake-3.8 patch, since the one kensington added some time ago seems to work as well.

Six (6) packages still have QA issues that I've not yet fully figured out, like so:

```
>>> Completed installing lxqt-session-0.11.1 into
 /var/tmp/portage/lxqt-base/lxqt-session-0.11.1/image/

 * Final size of build directory: 3048 KiB
 * Final size of installed tree: 612 KiB

 * QA Notice: This package installs one or more .desktop files that do not
 * pass validation.
 *
 *      /usr/share/applications/lxqt-config-session.desktop: error: value
    "Settings;DesktopSettings;Qt;LXQt;" for key
    "Categories" in group "Desktop Entry" contains an
    unregistered value "LXQt"; values extending the format should start with "X-"
```

**Comment by Chiitoo** [comment]
While working on live ebuilds for these, I realised that I may have misunderstood what repoman was telling me about the '~arm' keyword, and I should probably restore those before these are merged...

That said, there may be new releases around the corner again/finally, too, so these may be out of date soon anyblue.

**Comment by Chiitoo** [comment]
Indeed, 0.12.0 is finally being released as I type this, so I will be updating this bump to reflect that at once I procure the time for it.  :]

**Comment by a17r** [comment]
> Six (6) packages still have QA issues that I've not yet fully figured out, like so:

Seems you are not the only one: https://github.com/lxde/lxqt/issues/155#issuecomment-229049531

**Comment by Chiitoo** [comment]
Indeed...

I may have stumbled upon that when looking into the issue previously.

I also seem to remember a thread over at our forums, possibly Lumina related, that was going through the same issues.

Of course I can't find it right now.

**Comment by Chiitoo** [comment]
Me slacking for too long, or long enough.

Obsolete now due to the changes made in [Lxqt updates #5806](https://github.com/gentoo/gentoo/pull/5806).

**Pull request closed without merging**